client/client.cpp

```diff
@@ -186,10 +192,6 @@ bool Client::receive(std::vector<std::string>& updates) {
     error_ = "No active connection";
     return false;
   }
-  if (!conn_->poll(30000)) {
-    error_ = "Timeout while waiting for server";
-    return false;
-  }
```

Comment:
I concur with Vasil, things that can be blocking can be hung because we're debugging / attached for instance. We do not want to have a timeout occur there.